sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -623,24 +662,26 @@ public virtual async Task<Response<KeyVaultCertificateWithPolicy>> RestoreCertif
         }
 
         /// <summary>
-        /// Imports a pre-existing certificate to the key vault. The specified certificate must be in PFX or PEM format, and must contain the private key as well as the x509 certificates. This operation requires the
+        /// Imports a pre-existing certificate to the key vault. The specified certificate must be in PFX or PEM format, and must contain the private key as well as the X.509 certificates. This operation requires the
```

Comment:
Could you provide links to info on PFX/PEM formats?

Reply:
In this case, I don't know of a definitive source we should reference. A simple search of either term, though, presents lots of great information.